sdcm/provision/common/configuration_script.py

```diff
@@ -74,6 +75,7 @@ def _script_body(self) -> str:
         # 4. There is race condition between sct and boot script, disable ssh to mitigate it
         # 5. Make sure that whenever you use "cat <<EOF >>/file", make sure that EOF has no spaces in front of it
         script = ''
+        script += disable_daily_apt_triggers()
```

Comment:
Check for ubuntu or debian?

Reply:
There is an internal check for apt-get existence, that would be enough

Reply:
Keep in mind we create this script before a node is created (i.e. cloud-init)

Reply:
yes, we don't know underlying OS at this stage.